pkg/mcp/source/source.go

```diff
@@ -38,6 +38,7 @@ var scope = log.RegisterScope("mcp", "mcp debugging", 0)
 // be used with the mcp.MeshConfigRequest and mcp.RequestResources. It can
 // be removed once we fully cutover to mcp.RequestResources.
 type Request struct {
+	PeerAddr   string
```

Comment:
I don't think we should add this. This is a transitional data structure, before it becomes subsumed by the proto:
https://github.com/istio/api/blob/bd778017b4be1632f9d8bb23de3ace8b1d0d1f74/mcp/v1alpha1/mcp.proto#L199

Reply:
@ozevren Thanks for your comment. I tried to get the `peerAddr` from `SinkNode`. but there is no such information. the `Id` of `SinkNode` is "" for pilot and is `default` for mixer. Could you point me out how to get the peerAddr? Thanks.

Reply:
First, let's add Id as well to the UI.
My comment is about expanding the Request object. We should not do this. Otherwise we won't be able to get rid of it later. The address will probably needs to be carried through alongside the Request, and not within it.

Reply:
@ozevren I have added a new parameter to pass through the address.